README.Rmd

```diff
@@ -18,51 +18,137 @@ knitr::opts_chunk$set(
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
 <!-- badges: end -->
 
-{{ packagename }} provides functions to ....
+`{simulist}` is an R package to simulate infectious disease outbreak data, including line lists and contacts.
 
-<!-- This sentence is optional and can be removed -->
-{{ packagename }} is developed at the [CENTER|similar](url) at the [UNIVERSITY|similar](url) as part of the [Epiverse-TRACE program](https://data.org/initiatives/epiverse/).
+`{simulist}` is developed at the [Centre for the Mathematical Modelling of Infectious Diseases](https://www.lshtm.ac.uk/research/centres/centre-mathematical-modelling-infectious-diseases) at the [London School of Hygiene and Tropical Medicine](https://www.lshtm.ac.uk/) as part of [Epiverse-TRACE](https://data.org/initiatives/epiverse/).
 
 ## Installation
 
-You can install the development version of {{ packagename }} from
+You can install the development version of `{simulist}` from
 [GitHub](https://github.com/) with:
 
 ``` r
-# install.packages("pak")
-pak::pak("{{ gh_repo }}")
+# check whether {pak} is installed
+if(!require("pak")) install.packages("pak")
+pak::pak("epiverse-trace/simulist")
 ```
 
-## Example
+## Quick start
 
-These examples illustrate some of the current functionalities
+```{r load-simulist}
+library(simulist)
+library(epiparameter)
+```
+
+The line list simulation requires a serial interval, onset-to-hospitalisation delay, and onset-to-death delay. We can load these load these from the library of epidemiological parameters in the `{epiparameter}` R package, or if these are not available, such as the serial interval for COVID-19 we can create them ourselves.
 
+```{r create-epidists}
+# create COVID-19 serial interval
+serial_interval <- epiparameter::epidist(
+  disease = "COVID-19",
+  epi_dist = "serial interval",
+  prob_distribution = "gamma",
+  prob_distribution_params = c(shape = 1, scale = 1)
+)
 
-## Development
+# get onset to hospital admission from {epiparameter} database
+onset_to_hosp <- epiparameter::epidist_db(
+  disease = "COVID-19",
+  epi_dist = "onset to hospitalisation",
+  single_epidist = TRUE
+)
+
+# get onset to death from {epiparameter} database
+onset_to_death <- epiparameter::epidist_db(
+  disease = "COVID-19",
+  epi_dist = "onset to death",
+  single_epidist = TRUE
+)
+```
 
-### Lifecycle
+To simulate a line list for COVID-19 with an assumed reproduction number (`R`) of 1.1 we use the `sim_linelist()` function.
 
-This package is currently a *concept*, as defined by the [RECON software
-lifecycle](https://www.reconverse.org/lifecycle.html). This means that essential
-features and mechanisms are still being developed, and the package is not ready
-for use outside of the development team.
+```{r sim-linelist}
+linelist <- sim_linelist(
```

Comment:
I initially got this error:
```
Error in rep(times, next_gen) + serial(sum(n_offspring)) : 
  non-numeric argument to binary operator
```
Then realised it was issue with `bpmodels` dependency. Should we make this clearer to user (i.e. that the function is calling a simulation model from elsewhere)? And presumably will want to switch to `epichains` now `bpmodels` is deprecated?